### PR TITLE
Fixed interwiki prefixes, added shorthands and tests

### PR DESCRIPTION
**Pull request by takidelfin**
#2128

Fixed interwiki prefixes by reordering them (as stated in issue), added shorthands from [en.wikipedia.org/wiki/Help:Interwiki_linking](https://en.wikipedia.org/wiki/Help:Interwiki_linking#Project_titles_and_shortcuts). I also wrote tests for `lib/wikitext.rb`

GCI Task

### CODE REVIEW

File: spec/lib/wikitext_spec.rb [hunk 1]
```diff
@@ -84,28 +84,73 @@
...
-      output = subject.assignments_to_wikilinks(assignments, en_wiki)
-      expect(output).to include('[[Selfie]], ')
-      expect(output).to include('[[:Category:Photography]]')
-      expect(output).to include('[[Bishnu Priya]]')
-      expect(output).to include('[[:es:Blanca de Beaulieu]]')
-      expect(output).to include('[[:es:wiktionary:agrazarías]]')
-      expect(output).to include('[[wikidata:Q60]]')
+      VCR.use_cassette('multiwiki_assignment') do
+        output = subject.assignments_to_wikilinks(assignments, en_wiki)
+        puts output
```
Comment (psinghal20) [inline spec/lib/wikitext_spec.rb]: I don't think we should log any output while running ruby specs.

**Comment by takidelfin** [comment]
@ragesoss can you review my pull request?

File: lib/wikitext.rb [hunk 2]
```diff
@@ -64,12 +64,28 @@ def self.format_assignment_title(assignment, home_wiki)
     project = assignment.wiki.project
     language = assignment.wiki.language
 
+    # Interwiki shorthands, source: https://en.wikipedia.org/wiki/Help:Interwiki_linking#Project_titles_and_shortcuts
+    shorthands = {
```
Comment (ragesoss) [inline lib/wikitext.rb]: Instead of defining this in the method, this hash should be a constant.

File: spec/lib/wikitext_spec.rb [hunk 3]
```diff
@@ -84,15 +84,46 @@
...
+    let(:pl_wikiquote) { create(:wiki, language: 'pl', project: 'wikiquote') }
+    let(:es_wikisource) { create(:wiki, language: 'es', project: 'wikisource') }
     let(:wikidata) { create(:wiki, project: 'wikidata') }
+    let(:de_wikiversity) { create(:wiki, language: 'de', project: 'wikiversity') }
+    let(:en_wikivoyage) { create(:wiki, language: 'en', project: 'wikivoyage') }
+    # Disabled due to Invalid Project:
+    # (`raise InvalidWikiError unless PROJECTS.include?(project)`) in app/models/wiki.rb
+    # let(:metawikipedia) { create(:wiki, language: 'en', project: 'metawikipedia') }
+    # let(:meta) { create(:wiki, language: 'en', project: 'meta') }
+    # let(:commons) { create(:wiki, language: 'en', project: 'commons') }
```
Comment (ragesoss) [inline spec/lib/wikitext_spec.rb]: Commons is a valid project, it's just done as `language: 'commons', project: 'wikimedia'`

Comment (takidelfin) [reply spec/lib/wikitext_spec.rb]: what do to with `Commons => c` shorthand then? should I do something like ` shorthands[language_prefix] || language_prefix`?

**Pull request merged**